packages/kbn-pm/src/commands/build_bazel.ts

```diff
@@ -6,17 +6,14 @@
  * Side Public License, v 1.
  */
 
-import { runBazel } from '../utils/bazel';
-import { ICommand } from './';
+import { log } from '../utils/log';
+import { ICommand } from '.';
 
 export const BuildBazelCommand: ICommand = {
-  description: 'Runs a build in the Bazel built packages',
+  description: 'Removed',
   name: 'build-bazel',
 
   async run(projects, projectGraph, { options }) {
-    const runOffline = options?.offline === true;
-
-    // Call bazel with the target to build all available packages
-    await runBazel(['build', '//packages:build', '--show_result=1'], runOffline);
+    log.error('Command "build-bazel" has been replaced with "build"');
```

Comment:
Thoughts on how we want to handle this?  It's going to list build-bazel in the list of valid commands still.  Not a huge issue.

Reply:
I think if you set a reminder to remove this in a couple months we should be fine

Reply:
Actually, this has only been a valid command for like a month, maybe we can just drop it as I doubt anyone really got used to using it.